### DISCUSSION TITLE
Core team sync 2024-05-13

### DISCUSSION THREAD

**Discussion by madolson** [valkey-io/valkey#535] [General]
* Branding
    * should we reserve valkey-* for official valkey projects like clients and modules?
        * @PingXie will send an email about trademark.
        * Punting the decision until we have more information.
* Modules, https://github.com/valkey-io/valkey/issues/408
    * How are we bundling modules?
        * Should JSON be included by default?
        * We will distribute modules as two different flavors.
    * JSON - how to move forward? timeline? etc
        * Option 1: If we can support it, we should.
        * Option 2: How complex the code is, how many new libraries would we need to vendor.
        * Option 3: Core data platform. 
     * Clients should support all "core" commands?
        * Need to understand the line for what is core and what is not core?
        * No clear consensus yet
* Replica redirects - https://github.com/valkey-io/valkey/pull/325
    * What are we doing about it being a breaking change.
        * Update the change to have the client indicate support instead of a server configuration.
* Valkey.io website
    * Status, when will all the content be up?
    * Zola PR is in progress and is close.

**Comment by davefojtik** [comment]
Very happy to see the JSON being actively discussed! Hopefully, we'll get a repo for it on the upcoming meeting 😄
